src/test/java/org/takes/rq/RqMultipartTest.java

```diff
@@ -85,4 +179,75 @@ public void parsesHttpBody() throws IOException {
         );
     }
 
+    /**
+     * RqMultipart can return empty iterator on invalid part request.
+     * @throws IOException If some problem inside
+     */
+    @Test
+    public void returnsEmptyIteratorOnInvalidPartRequest() throws IOException {
+        final Request req = create(
+            Joiner.on(CR).join(
+                "Content-Disposition: form-data; name=\"address\"",
+                "",
+                "443 N Wolfe Rd, Sunnyvale, CA 94085",
+                // @checkstyle LineLength (1 line)
+                "Content-Disposition: form-data; name=\"data\"; filename=\"a.zip\""
```

Comment:
@ekondrashev let's use new joiner here on `; ` character and avoid line length checkstyle suppression